genesyscloud/journey_segment/resource_genesyscloud_journey_segment_schema.go

```diff
@@ -52,6 +52,7 @@ var (
 			Required:     true,
 			ForceNew:     true, // scope can be only set during creation
 			ValidateFunc: validation.StringInSlice([]string{"Session"}, false),
+			Deprecated:   "The scope field is redundant and will be removed in a future release.",
```

Comment:
One more small thing - I'm pretty sure you'll have to re-run `make docs` to render this change